docs/install-w-machine.md

```diff
@@ -84,6 +84,11 @@ Daemon running on each node. Other discovery service backends such as
 	The `swarm create` command returned the  `fe0cc96a72cf04dba8c1c4aa79536ec3`
 	token.
 
+	**Note:** This command relies on Docker Swarm's hosted discovery service. If
```

Comment:
@ahaurw01 Thanks for the contribution. The Note is formatted incorrectly.  Can you fix that please

```markdown
>**Note**: This command relies on Docker Swarm's hosted discovery service. If
this service is having issues, this command may fail. In this case, see
information on using other types of [discovery backends](./discovery). Check
the [status page](http://status.docker.com/) for service availability.
```